tests/ci/run_s2n_integration.sh

```diff
@@ -19,6 +19,7 @@ cmake ${AWS_LC_DIR} -GNinja "-B${ROOT}/aws-lc-build" "-DCMAKE_INSTALL_PREFIX=${R
 ninja -C aws-lc-build install
 
 # While AWS-LC is a private project pretend to be BoringSSL so s2n uses the correct APIs
```

Comment:
We should change this wording. We are not private anymore.

Reply:
This will be removed when it's ready to execute the `TODO`, which is in line 22.